templates/components/entry.html.twig

```diff
@@ -78,7 +78,16 @@
             {% endif %}
             <aside class="meta entry__meta">
                 <span>{{ component('user_inline', {user: entry.user, showAvatar: SHOW_USER_AVATARS is same as V_TRUE}) -}}</span>
-                <span>,</span>
+                {% if entry.user.admin() %}
+                    <span class="user-badge">{{ 'user_badge_admin'|trans }}</span>
+                {% elseif entry.user.moderator() %}
+                    <span class="user-badge">{{ 'user_badge_global_moderator'|trans }}</span>
+                {% elseif entry.magazine.userIsModerator(entry.user) %}
+                    <span class="user-badge">{{ 'user_badge_moderator'|trans }}</span>
+                {% elseif (entry.user.type) == "Service" %}
+                    <span class="user-badge">{{ 'user_badge_bot'|trans }}</span>
+                {% endif %}
```

Comment:
This sounds like the bot badge is not shown if the user is one of the previous. But it imo it should be shown

Reply:
changed